betterdiscordctl

```diff
@@ -430,161 +324,63 @@ bdc_find_modules() {
 bdc_snap() {
   # shellcheck disable=SC2016
   # Expansion should happen inside snap's shell.
-  snap_location=$("$snap_bin" run --shell discord <<< $'printf -- \'%s\n\' "$SNAP" 1>&3' 3>&1)
-  discord=${snap_location:?}/usr/share/discord
-  verbose 2 "VV: Checking $discord"
-  if [[ -d $discord ]]; then
-    verbose 1 "V: Using Discord at $discord"
-    # shellcheck disable=SC2016
-    # Expansion should happen inside snap's shell.
-    xdg_config=$("$snap_bin" run --shell discord <<< $'printf -- \'%s/.config\n\' "$SNAP_USER_DATA" 1>&3' 3>&1)
-    discord_config=$xdg_config/discord
+  xdg_config=$("$snap_bin" run --shell discord <<< $'printf -- \'%s/.config\n\' "$SNAP_USER_DATA" 1>&3' 3>&1)
+  discord_config=$xdg_config/discord
+  if [[ -d $discord_config ]]; then
     bdc_find_modules
   else
     die 'ERROR: Discord installation not found.'
```

Comment:
Should be style `Config directory not found for Discord`, as it's no longer dying from lack of an overall installation.